Lib/bdb.py

```diff
@@ -331,6 +332,10 @@ def set_trace(self, frame=None):
         while frame:
             frame.f_trace = self.trace_dispatch
             self.botframe = frame
+            # Remember f_trace_lines of the frame, and set it to True
```

Comment:
The first line is a direct description of the code, which we don't need. The second line adds the reason, so we do need it.
```suggestion
```